src/com/lilithsthrone/game/Game.java

```diff
@@ -2788,6 +2842,10 @@ public boolean banishNPC(NPC npc) {
 	
 	public void banishNPC(String id) {
 		NPC npc = (NPC) getNPCById(id);
+		if(Main.game.getActiveNPC() == npc)
```

Comment:
Please, when you see stuff like this, don't add to the mess!

If you route banishNPC(String id) to banishNPC(NPC npc), then you don't have to add your code twice.

Reply:
Routed, fair enough.